targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F413xH/device/TOOLCHAIN_ARM_STD/stm32f413xh.sct

```diff
@@ -27,15 +28,24 @@
 ; OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
-; STM32F413ZH: 1536 KB FLASH (0x150000) + 320 KB SRAM (0x50000)
-LR_IROM1 0x08000000 0x150000  {    ; load region size_region
+#if !defined(MBED_APP_START)
+  #define MBED_APP_START 0x08000000
+#endif
 
-  ER_IROM1 0x08000000 0x150000  {  ; load address = execution address
+; 1536KB FLASH (0x150000) 
```

Comment:
1536K = 0x180000 ...

Reply:
:+1: apprently it never bothered anybody ....

Reply:
Good catch 🎉